src/DotNetWorker.Grpc/StatusResult.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Grpc.Messages
+{
+    internal sealed partial class StatusResult
+    {
+        private static StatusResult? _successInstance;
```

Comment:
Exposing this to reduce allocations. This prevents an instance creation for every single successful invocation (which accounts for the vast majority of them).